### PR TITLE
FEAT: Add Space Emergency Kit

### PR DESCRIPTION
**Pull request by SebaSphere**
Still needs a proper recipe. Potions aren't supported by default with the json recipe, so I need to add a new Recipe type that allows potions.

image

Using Uncraftable potion as placeholder. You can find actual recipe here: https://twitter.com/radfast/status/967841494928429057

It's a healing potion and long night vision potion.

### CODE REVIEW

File: src/main/java/com/hrznstudio/galacticraft/items/EmergencyKitItem.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        user.giveItemStack(new ItemStack(GalacticraftItems.SMALL_OXYGEN_TANK, 2));
+        user.giveItemStack(new ItemStack(GalacticraftItems.RED_PARACHUTE));
+        user.giveItemStack(new ItemStack(GalacticraftItems.CANNED_DEHYDRATED_POTATO));
+        user.giveItemStack(PotionUtil.setPotion(new ItemStack(Items.POTION), Potions.HEALING));
+        user.giveItemStack(PotionUtil.setPotion(new ItemStack(Items.POTION), Potions.LONG_NIGHT_VISION));
+
+    }
+
+
+}
```
Comment (marcus8448) [inline src/main/java/com/hrznstudio/galacticraft/items/EmergencyKitItem.java:57]: This awkward whitespace bothers me.

File: src/main/resources/assets/galacticraft-rewoven/lang/en_us.json [hunk 2]
```diff
@@ -319,6 +319,7 @@
   "item.galacticraft-rewoven.evolved_zombie_spawn_egg": "Evolved Zombie Spawn Egg",
   "item.galacticraft-rewoven.throwable_meteor_chunk": "Throwable Meteor Chunk",
   "item.galacticraft-rewoven.hot_throwable_meteor_chunk": "Hot Throwable Meteor Chunk",
+  "item.galacticraft-rewoven.emergency_kit": "Emergency Space Kit",
```
Comment (marcus8448) [inline src/main/resources/assets/galacticraft-rewoven/lang/en_us.json:322]: Maybe just `Emergency Kit`? (What does GC4 say)

**Review by marcus8448 — CHANGES_REQUESTED** [review]
License headers and small things

File: src/main/java/com/hrznstudio/galacticraft/items/EmergencyKitItem.java (hunk 1 above)
Comment (craftablescience) [reply src/main/java/com/hrznstudio/galacticraft/items/EmergencyKitItem.java:57]: See @SebaSphere it's not just me

File: src/main/resources/assets/galacticraft-rewoven/lang/en_us.json (hunk 2 above)
Comment (SebaSphere) [reply src/main/resources/assets/galacticraft-rewoven/lang/en_us.json:322]: GC4 actually says "Space Emergency Kit"
https://github.com/micdoodle8/Galacticraft/blob/9f80b35bc06e9942e4e2b646bd91710a3f2586ff/src/main/resources/assets/galacticraftcore/lang/en_US.lang#L469

Do we plan on sticking with GC4's naming convention or shorten it to `Emergency Kit`

File: src/main/java/com/hrznstudio/galacticraft/items/EmergencyKitItem.java (hunk 1 above)
Comment (SebaSphere) [reply src/main/java/com/hrznstudio/galacticraft/items/EmergencyKitItem.java:57]: I'm terrible with spacing my whitespace but yeah I'll fix that

File: src/main/resources/assets/galacticraft-rewoven/lang/en_us.json (hunk 2 above)
Comment (JoeZwet) [reply src/main/resources/assets/galacticraft-rewoven/lang/en_us.json:322]: `Emergency Kit` feels more natural to me

Comment (vos6434) [reply src/main/resources/assets/galacticraft-rewoven/lang/en_us.json:322]: +1 for Emergency Kit

**Comment by SebaSphere** [comment]
Got the time to work on this again. Expect a PR sometime in the upcoming days 😃

**Comment by craftablescience** [comment]
You might want to rebase on `main` before doing any more changes btw, that way less conflicts

**Comment by SebaSphere** [comment]
Will be closing PR. Currently working on this in the `feat/emergency_kit` branch and will make a new PR when done.

**Comment by JoeZwet** [comment]
Could just make it a draft pr

**Pull request closed without merging**